dataService.js

```diff
@@ -0,0 +1,77 @@
+// All Maps declaration
+let productMap;
+let categoryMap;
+let combinedProductMap;
+
+// It starts products functions
+function getProducts() {
+
+  if (productMap) {
+    return productMap;
+  };
+  // Otherwise, load following Map for all the defined products
+  const productsRawData = require('./src/data/products.json');
+  const productMapForReturn = new Map();
+
+  for (const food of productsRawData.products) {
```

Comment:
How about 
```suggestion
  for (const product of productsRawData.products) {
```
since `food` is not a good variable name in this context.

Reply:
Done